src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="field">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text"name="surname"
+          required
+          >
```

Comment:
If you have more than 2 attributes - **each of them** should be on a new line and have **exactly two indents**
```suggestion
          <input 
            type="text"
            name="surname"
            required
          >
```

Reply:
fixed. but i forgot to add DEMO links again... here they are
- [DEMO LINK](https://Dima-Petrov.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dima-Petrov.github.io/layout_html-form/report/html_report/)

Reply:
fixed